docs/docsite/rst/intro_installation.rst

```diff
@@ -82,10 +82,10 @@ default this uses sftp. If that's not available, you can switch to scp in
    Ansible 2.2 introduces a tech preview of support for Python 3. For more information, see `Python 3 Support <http://docs.ansible.com/ansible/python_3_support.html>`_.
 
    By default, Ansible uses Python 2 in order to maintain compatibility with older distributions
-   such as RHEL 6. However, some Linux distributions (Gentoo, Arch) may not have a
+   such as RHEL 6. However, some Linux distributions (Fedora, Arch, Gentoo) may not have a
```

Comment:
actually, i would change the whole paragraph as we are supporting python3 now, but should note that >=3.5